gradle.properties

```diff
@@ -23,7 +23,7 @@ org.gradle.caching=true
 org.gradle.jvmargs=-Xmx4g
 
 # version that is going to be updated automatically by releases
-version = 0.8.0-incubating-SNAPSHOT
+version = 0.7.0-incubating-SNAPSHOT
```

Comment:
please don't update these properties

Reply:
This is fixed now, thanks for the feedback

Reply:
revert pythonVersion too?

Reply:
oops sorry, that's done